core/services/pipeline/runner.go

```diff
@@ -250,6 +250,8 @@ func (r *runner) executeRun(ctx context.Context, txdb *gorm.DB, spec Spec, meta
 		}
 	}
 
+	vars := Vars{}
+
 	// TODO: Test with multiple and single null successor IDs
```

Comment:
Did this TODO ever get done?

Reply:
This is the first low-level pipeline work I've done since before this TODO was written